rmf_obstacle_ros2/src/lane_blocker/LaneBlocker.cpp

```diff
@@ -26,7 +26,7 @@
 #include <geometry_msgs/msg/pose_stamped.hpp>
 #include <geometry_msgs/msg/vector3_stamped.hpp>
 
-#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
```

Comment:
Please undo this change. `Galactic` version of `tf2_geometry_msgs` only contains `.h` header while `Humble` contains both. So until we move to I turtle, the original is fine and will enable us to build on both `Galactic` and `Humble`.

Reply:
my bad, i didnt realize that!